pkg/modules/metricsmodule/implmetricsmodule/handler.go

```diff
@@ -0,0 +1,155 @@
+package implmetricsmodule
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/SigNoz/signoz/pkg/errors"
+	"github.com/SigNoz/signoz/pkg/http/binding"
+	"github.com/SigNoz/signoz/pkg/http/render"
+	"github.com/SigNoz/signoz/pkg/modules/metricsmodule"
+	"github.com/SigNoz/signoz/pkg/types/authtypes"
+	"github.com/SigNoz/signoz/pkg/types/metricsmoduletypes"
+	"github.com/SigNoz/signoz/pkg/valuer"
+	"github.com/gorilla/mux"
+)
+
+type handler struct {
+	module metricsmodule.Module
+}
+
+// NewHandler returns a metricsmodule.Handler implementation.
+func NewHandler(m metricsmodule.Module) metricsmodule.Handler {
+	return &handler{
+		module: m,
+	}
+}
+
+func (h *handler) GetStats(rw http.ResponseWriter, req *http.Request) {
+	claims, err := authtypes.ClaimsFromContext(req.Context())
+	if err != nil {
+		render.Error(rw, err)
+		return
+	}
+
+	var in metricsmoduletypes.StatsRequest
+	if err := binding.JSON.BindBody(req.Body, &in); err != nil {
+		render.Error(rw, err)
+		return
+	}
+
+	orgID, err := valuer.NewUUID(claims.OrgID)
+	if err != nil {
+		render.Error(rw, err)
+		return
+	}
```

Comment:
```suggestion
	orgID := valuer.MustNewUUID(claims.OrgID)
```

Always remember, for claims, you can call `MustNewUUID`.

Reply:
done